src/components/date-picker.js

```diff
@@ -1071,18 +1085,22 @@ const DatePicker = ({
               </Menu.Button>
               :
               <>
-                {label && <div className='mb-1.5 flex flex-row items-center'>
-                  <p className='mr-1 font-pt text-xs text-secondary-600 tracking-md leading-1.33'>{label}</p>
+                {label.title && <div className={labelClasses.container}>
+                  <p className={labelClasses.title}>{label.title}</p>
```

Comment:
this would be a breaking change, anyway to make it backwards compatible? (e.g. maybe still accept label as a string along with `label.title`)

Reply:
I think the only currently project that is using it is ql at the moment but yea sure I can do it backwards compatible

Reply:
summarizing our convo here for reference:
- would like a fallback to a `label` string not just for backwards compatibilities, also for ease of use --> most of the times `label` will likely be a string, hence giving the prop value `label='some string'` is more convenient than `label={{ title: 'some string' }}`, we can reserve the object version of `label` for more complex usages, such as ones that needs a custom icon and tooltip
- does not necessarily need additional props, checking if `label` is an object or string would be sufficient

thanks again for the changes! 🙏